bucket/musescore.json

```diff
@@ -3,7 +3,7 @@
     "homepage": "https://musescore.org/",
     "version": "3.0.4.5763",
     "license": "GPL-2.0-only",
-    "url": "https://ftp.osuosl.org/pub/musescore/releases/MuseScore-3.0.4/MuseScore-3.0.4.msi",
+    "url": "https://github.com/musescore/MuseScore/releases/download/v3.0.4/MuseScore-3.0.4-x86_64.msi",
```

Comment:
Thanks for updating the URL, but this change will be lost the next time the manifest is updated. You need to make the corresponding change to the `autoupdate` URL further down the file. See [App Manifest Autoupdate](https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate).

The autoupdate URL should be:

```json
    "autoupdate": {
        "url": "https://github.com/musescore/MuseScore/releases/download/v$matchHead/MuseScore-$matchHead-x86_64.msi",
```

Reply:
Thanks for making the change!